src/main/java/com/hiqes/andele/Andele.java

```diff
@@ -765,6 +842,8 @@ public void handleMessage(Message msg) {
                     //  education info about, so do the request.
                     if (!skipAsk) {
                         doRequest(msg.arg1);
+                    } else {
+                        sReqMgr.removeRequest(msg.arg1);
```

Comment:
Add a comment describing why the request is being removed.

Reply:
Done